scripts/create_singularities

```diff
@@ -37,22 +37,22 @@ IMAGES_DIR="images"
 
 
 function debug() {
-	: # echo -e "D: $@" >&2
+	: # echo -e "D:" "$@" >&2
 }
 
 function info() {
-	echo -e "I: $@" >&2
+	echo -e "I:" "$@" >&2
 }
 
 function error() {
-	echo -e "E: $@" >&2
+	echo -e "E:" "$@" >&2
```

Comment:
I was not 100% certain that separating this way wouldn't cause some values to be interpreted as options to echo, but seems to be not:

```shell
$> cat /tmp/checkecho
#!/bin/bash
echo -e "I1: $@"
echo -e "I2:" "$@"

$> /tmp/checkecho '-n' 1 "2 3" '&' '${var}'
I1: -n 1 2 3 & ${var}
I2: -n 1 2 3 & ${var}
"""
so all good